plugin/src/main/kotlin/me/tylerbwong/gradle/metalava/extension/MetalavaExtension.kt

```diff
@@ -114,6 +114,17 @@ open class MetalavaExtension @Inject constructor(
      */
     val enforceCheck: Property<Boolean> = objectFactory.property { set(true) }
 
+    /**
+     * If the signature task should also generate keep rules. The default is false.
+     */
+    val generateKeepFile: Property<Boolean> = objectFactory.property { set(false) }
```

Comment:
I would get rid of this if possible and just include generating the keep file if the path is not null. Default to null.

Reply:
That's fair. I think that's the way to go to keep user configuration as simple as possible.